src/rpc/mining.cpp

```diff
@@ -566,6 +566,8 @@ static RPCHelpMan getblocktemplate()
                     {"segwit", RPCArg::Type::STR, RPCArg::Optional::NO, "(literal) indicates client side segwit support"},
                     {"str", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "other client side supported softfork deployment"},
                 }},
+                {"longpollid", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "Delay processing request until the result would vary significantly from the \"longpollid\" of a prior template."},
```

Comment:
1. This should not have a period '.' at the end. Since it is inconsistent with the rest of the documentation. 
2. Should start with a lowercase letter since only grammatical articles have capitalized first letters for this documentation.  (A, The, This)
3. The language used in BIP22 is a more concise definition ```"longpollid" of job to monitor for expiration; required and valid only for long poll requests```